torch/csrc/cuda/Module.cpp

```diff
@@ -541,8 +541,25 @@ PyMethodDef* THCPModule_methods() {
 
 namespace torch { namespace cuda {
 
+namespace shared {
+
+void initCudartBindings(PyObject* module);
+void initNvtxBindings(PyObject* module);
+#if defined(USE_CUDNN) || defined(__HIP_PLATFORM_HCC__)
+void initCudnnBindings(PyObject* module);
+#endif
+
+} // namespace shared
+
 void initModule(PyObject *module) {
   python::initCommMethods(module);
+  // As weird as it seems, this file is also compiled for ROCm,
+  // so this condition might not always be true...
+  shared::initCudartBindings(module);
+  shared::initNvtxBindings(module);
+#ifdef USE_CUDNN
+  shared::initCudnnBindings(module);
```

Comment:
@peterbell10 It appears that `initCudnnBindings` was intended to be defined and called for both cuDNN and MIOpen based on the declaration above, but is only called for cuDNN here. Was this intentional?

Reply:
Oops, your right. That should be 
```cpp
#if defined(USE_CUDNN) || defined(__HIP_PLATFORM_HCC__)
```

Reply:
Will you be submitting the change in a different PR/commit, or would you like me to make that a part of #33837?

Reply:
I don't have a ROCm system to test on so if you want to take over, that would be great.